copier.yml

```diff
@@ -69,10 +69,10 @@ RunJuliaNightlyOnCI:
   type: bool
   help: Package tests run on Julia version {{ JuliaMinVersion }} and on the latest stable release. Do you also want to run them on the nightly version?
 
-TestPRsOnlyOneMachine:
+SimplifiedPRTest:
   when: "{{ AskAdvancedQuestions }}"
   type: bool
-  help: Do you want to limit the PR test to a single machine (ubuntu + latest stable Julia)? This will make the PR tests run faster, but might introduce OS-specific or LTS specific errors.
+  help: (Simplified PR Test) Do you want to limit the Pull Request test to a single configuration (ubuntu + latest stable Julia)? This will make the PR tests run faster, but might introduce OS-specific or LTS specific errors.
```

Comment:
```suggestion
  help: (Simplified PR Test) Do you want to limit the Pull Request test to a single configuration (ubuntu + latest stable Julia)? This will make the PR tests run faster, but might miss OS-specific or LTS specific errors.
```